src/Bonnier/WP/Cxense/Assets/Scripts.php

```diff
@@ -79,6 +79,11 @@ public function head_tags()
                 $recs_tags[$this->org_prefix . 'taxo-tag'] = $this->objects_to_array(get_the_tags());
             }
 
+            if ($commercialType = get_field('commercial_type', $post->ID)) {
```

Comment:
This logic couples the wp-cxense plugin to closely together with the contenthub-editor plugin

It should instead be added to the gds-frontend project by using the filter: 
```add_filter('cxense_head_tags', function($recs_tags){
 // do your magic here
});```